UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs

```diff
@@ -449,6 +453,16 @@ public bool Interact(HandActivate interaction)
 			return true;
 		}
 
+		foreach (var slot in slots)
+		{
+			if(slot.IsEmpty) continue;
+			if (slot.ItemObject.TryGetComponent<MouseTrap>(out var trap))
+			{
+				if(trap.IsArmed) trap.TriggerTrapFromContainer(interaction.Performer.GetComponent<LivingHealthMasterBase>());
+				break;
+			}
+		}
```

Comment:
Just noticed this Interact is ClientSide only, or at least line 426 says so. So this will need to be moved?

Reply:
Humm, i dont think server knows when a player has opened a bag. So here you'll need to call a command in PlayerNetworkedActions which'll trigger the trap inside the bag

Reply:
like this?

Reply:
https://github.com/unitystation/unitystation/pull/7743/commits/10ef13bf2430ef7803aa5ab000f484c5a59b97f7